keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -478,46 +486,49 @@ Because Windows privileged containers will work much differently than Linux priv
-- Named Pipe mounts will **not** be supported. Instead named pipes should be accessed via their path on the host (\\\\.\\pipe\\*).
-  - The following error will be returned if `hostProcess` containers attempt to use name pipe mounts - https://github.com/microsoft/hcsshim/blob/358f05d43d423310a265d006700ee81eb85725ed/internal/jobcontainers/mounts.go#L40.
-- Unix domain sockets mounts support will be added before `HostProcess` containers graduate to `stable`. For `alpha` and `beta` Unix domain sockets can be accessed via their paths on the host like named pipes.
-  - The Windows APIs needed to support mounting unix domain socket mounts in `hostProcess` containers was introduced in Windows Server Ver, 2004. Microsoft is planning on backporting these APIs to Windows Server 2019 (min support Windows Server OS) to provide a consistent user experience.
-- Mounting directories from the host OS into `hostProcess` containers will work just like with normal containers - kubelet will explicitly block this scenario.
-- All other volume types supported for normal containers on Windows will work with `hostProcess` containers.
+- Window's bind-filter driver will be used to create a view that merges the host's OS filesystem with container-local volumes.
+When `hostProcess` containers are started a new volume will be created which contains the contents of the contaner image.
+This volume will be mounted to `c:\hpc`. The default working directory for `hostProcess` containers will also be set to `c:\hpc`.
+- Volume mounts (includinge service account tokens) will be supported for `hostProcess` containers and can be accessed just the same way as regular Windows Server containers.
+  - Named Pipe mounts will **not** be supported.
+    Instead named pipes should be accessed via their path on the host (\\\\.\\pipe\\*).
+    The following error will be returned if `hostProcess` containers attempt to use name pipe mounts -
+    https://github.com/microsoft/hcsshim/blob/358f05d43d423310a265d006700ee81eb85725ed/internal/jobcontainers/mounts.go#L40.
+  - Unix domain sockets mounts also not not be supported for `hostProces` containers.
+    Unix domain sockets can be accessed via their paths on the host like named pipes.
+  - Mounting directories from the host OS into `hostProcess` containers will work just like with normal containers but this is not recommend.
+    Instead workloads should access the host OS's file-system as if it was not being run in a container.
+  - All other volume types supported for normal containers on Windows will work with `hostProcess` containers.
+- `HostProcess` Containers will have full access to the host file-system (unless restricted by filed-based ACLs and the run_as_username used to start the container).
+- There will be no `chroot` equivalent.
+
+- Note: Behavior of volume mounts will differ between the alpha/beta (old) implementation of this feature and the stable (new) implementation.
```

Comment:
Can you summarize the differences here and how maintaining backwards compatibility is planned? I know it's detailed in the links below, but I think it would be good to state it here.

Reply:
Yes, I will summarize the differences and also explain the backwards compatibility.

For backwards compatibility there is an annotation you can set for the pod that will use the old or the new behavior.

This must be set during pod sandbox creation and will apply to all containers in the pod.
If the new behavior is requested and the required APIs are not present on the machine (for Windows Server 2019) CreatePodSandbox CRI call will fail.

If the annotation is not set you'll get the new behavior is the required APIs are available and or the old behavior if they are not. 
Note: The APIs will be available for Windows Server 2019 in July 2022 and have been present in Windows Server 2022 since launch.

Reply:
@msau42 I made some updates to address your comments PTAL and thanks!